src/components/DevfileEditor/index.tsx

```diff
@@ -102,6 +102,13 @@ export class DevfileEditor extends React.PureComponent<Props, State> {
     const items = this.props.plugins.plugins;
     const components = jsonSchema && jsonSchema.properties ? jsonSchema.properties.components : undefined;
     if (components) {
+      const mountSources = components.items.properties.mountSources;
+
+      // TODO remove this temporary solution
```

Comment:
means there is some plan how to get rid of that... Do we really have that?
If there is no any plan - instead I would write a comment that explains why it's here, something like
> mount sources is specific only for some of component types but always appears  
> patch schema and remove default value for boolean mount sources to avoid their appearing during the completion

Reply:
Thanks. Done